Onelogin saml SCIM updates (#2246)
pulumi/pulumi-hugo: themes/default/content/docs/guides/saml/onelogin.md

1. Open the file and copy the entire block of XML text in your clipboard
1. Open the Pulumi Service and navigate to your SAML organization.

> 1. Select the **Access management** view, and then select **Change requirements**.

Reviewer: i think we should still say settings here, because they have to go to org settings... maybe...

```suggestion
1. Select **Access management** from organization settings, and then select **Change requirements**.
```

Author: I updated the onelogin saml and scim pages to match the verbage used on okta page.
